sway-fmt-v2/src/fmt.rs

```diff
@@ -855,4 +863,37 @@ impl Qux for Foo {
             Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
         assert_eq!(correct_sway_code, formatted_sway_code)
     }
+
+    #[test]
+    fn test_newline_sequence_formatting() {
+        let sway_code_to_format = r#"script;
+
+fn main() {
+    let number: u64 = 10;
+
+    let number2: u64 = 20;
+
+
+
+    let number3: u64 = 30;
+
+
+
+}"#;
+
+        let correct_sway_code = r#"script;
+
+fn main() {
+    let number: u64 = 10;
```

Comment:
Wouldn't we want to retain the extra space after this line? and if there are more lines than the threshold those extra lines would be removed?